geometry_msgs/Transform transform
int8[] occupancy_grid
int8[2] map_shape
int16[2] turtlebot_pos
int16[2] goal_pos
int16[2] marker_pos
int16[2] seg_img_shape
bool success